core/extension/gdextension_interface.h

```diff
@@ -1591,6 +1591,20 @@ typedef void (*GDExtensionInterfaceStringNewWithLatin1CharsAndLen)(GDExtensionUn
  */
 typedef void (*GDExtensionInterfaceStringNewWithUtf8CharsAndLen)(GDExtensionUninitializedStringPtr r_dest, const char *p_contents, GDExtensionInt p_size);
 
+/**
+ * @name string_new_with_utf8_chars_and_len2
+ * @since 4.3
```

Comment:
We should also add a `@deprecated` to the older `string_new_with_utf8_chars_and_len` function. Same goes for `strintg_new_with_utf16_chars_and_len`.

Reply:
Done.